### PR TITLE
chore(templates): determine http listener rule priority base on http listener

### PR DESCRIPTION
**Pull request by Lou1415926**
The problem with using https listener to determine the priority for `HTTPListenerRuleWithDomain` (a listener rule that's imposed on the HTTP listener, not HTTPS) is, when a service converts from HTTP to using HTTPS, the priority of `HTTPListenerRuleWithDomain` is not necessarily valid.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/template/templates/workloads/partials/cf/https-listener.yml [hunk 1]
```diff
@@ -102,7 +115,7 @@ HTTPListenerRuleWithDomain:
     {{- else}}
     ListenerArn: !GetAtt EnvControllerAction.HTTPListenerArn
     {{- end}}
-    Priority: !GetAtt HTTPSRulePriorityAction.Priority # Same priority as HTTPS Listener
+    Priority: !GetAtt HTTPRulePriorityAction.Priority # Same priority as HTTPS Listener
```
Comment (dannyrandall) [inline internal/pkg/template/templates/workloads/partials/cf/https-listener.yml]:
```suggestion
    Priority: !GetAtt HTTPRulePriorityAction.Priority
```

**Review by dannyrandall — APPROVED** [review]
great fix!

**Review by efekarakus — APPROVED** [review]
Huh how come this didn't cause a problem before?

**Comment by iamhopaul123** [comment]
> Huh how come this didn't cause a problem before?

Because there's no way for them to switch from http <-> https

**Pull request merged**